doc/classes/Joint2D.xml

```diff
@@ -8,6 +8,14 @@
 	</description>
 	<tutorials>
 	</tutorials>
+	<methods>
+		<method name="get_rid" qualifiers="const">
+			<return type="RID" />
+			<description>
+				Returns the object's [RID].
```

Comment:
```suggestion
				Returns the [RID] of the joint.
```
Or
```suggestion
				Returns joint's [RID].
```
Assuming this is copied from `CollisionObject2D`, where the word "object" is fitting